aspnet/SameSite/owin-samesite.md

```diff
@@ -76,7 +76,7 @@ The 2019 draft of the `SameSite` specification:
 ## Supporting older browsers
 
 The 2016 `SameSite` standard mandated that unknown values must be treated as `SameSite=Strict` values. Apps accessed from older browsers which support the 2016 `SameSite` standard may break when they get a `SameSite` property with a value of `None`. Web apps must implement browser detection if they intend to support older browsers. ASP.NET doesn't implement browser detection because User-Agents values are highly volatile and change frequently. An extension point in [ICookieManager](/previous-versions/aspnet/dn800238(v%3Dvs.113)) allows plugging in User-Agent specific logic.
-<!-- https://docs.microsoft.com/previous-versions/aspnet/dn800238(v%3Dvs.113) -->
+<!-- /previous-versions/aspnet/dn800238(v%3Dvs.113) -->
```

Comment:
Revert this change. It's in a comment where the complete URL is more convenient.